components/Page.js

```diff
@@ -0,0 +1,7 @@
+export default function Page({ children }) {
+  return (
+    <div className="max-w-7xl mx-auto flex flex-col p-6 align-center">
+      {children}
+    </div>
+  );
+}
```

Comment:
Would this be better to be added to the layout instead of a new component?

Reply:
The idea of a new component was to make sure all main pages remain consistent.

I think I did not understand where you would put it instead. In the two files in `components/layouts`?

Reply:
Ah ok, the problem I see there is that in that case it would also affect the homepage, adding unwanted spacing, since we're adding `mx-auto p-6` to the parent.

(in this line)
https://github.com/EddieHubCommunity/LinkFree/blob/ab10ce665ce4a155d07f0195f2d0d90a6adeb9d1/components/layouts/MultiLayout.js#L10

Causing something like this
![image](https://user-images.githubusercontent.com/7253929/206546016-4674763d-12a7-47e1-b30e-b12a578d5dff.png)

Reply:
ah good point 👍